### PR TITLE
Adds a simple util file which can be called to teardown database

### PR DESCRIPTION
**Pull request by darrellwarde**
# Description

Will need to be used to avoid hitting the 50,000 node limit when testing against Aura Free (would take a while, but would be a nuisance).

### CODE REVIEW

File: packages/graphql/tests/integration/teardown.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        console.log("Encountered an error whilst clearing down database!");
+        throw e;
+    } finally {
+        await session.close();
+        await driver.close();
+    }
+};
+
+// eslint-disable-next-line no-void
+void teardown().then(() => console.log("Successfully cleared down database."));
```
Comment (danstarns) [inline packages/graphql/tests/integration/teardown.ts:20]: Why void ?

**Review by danstarns — COMMENTED** [review]
This file isn't imported and so the function is never called?

**Comment by darrellwarde** [comment]
> This file isn't imported and so the function is never called?

No, it's to literally be called in TeamCity, `ts-node tests/integration/teardown.ts` before running tests against Aura, so we don't go over the 50,000 node limit. It's just a nice way to take advantage of the driver provided in `neo4j.ts`.

File: packages/graphql/tests/integration/teardown.ts (hunk 1 above)
Comment (darrellwarde) [reply packages/graphql/tests/integration/teardown.ts:20]: 🤣  Whatever you do here, either TypeScript or ESLint complains!!! This required the least comments in code.

**Comment by danstarns** [comment]
> > This file isn't imported and so the function is never called?
> 
> No, it's to literally be called in TeamCity, `ts-node tests/integration/teardown.ts` before running tests against Aura, so we don't go over the 50,000 node limit. It's just a nice way to take advantage of the driver provided in `neo4j.ts`.

Could we add a comment in this file for this ? I fear it is deleted in some random 'cleanup' in the future as someone doesn't know where it's being called from.

**Comment by darrellwarde** [comment]
> > > This file isn't imported and so the function is never called?
> 
> > 
> 
> > No, it's to literally be called in TeamCity, `ts-node tests/integration/teardown.ts` before running tests against Aura, so we don't go over the 50,000 node limit. It's just a nice way to take advantage of the driver provided in `neo4j.ts`.
> 
> 
> 
> Could we add a comment in this file for this ? I fear it is deleted in some random 'cleanup' in the future as someone doesn't know where it's being called from. 

Yeah good call, will do so in the morning! Only a temporary solution anyway, I'm hoping we can do something better around this in the near future. 🙂

**Review by oskarhane — APPROVED** [review]
+1 on @danstarns comment on adding a purpose comment to the file.
But other than that 👍

**Review by danstarns — COMMENTED** [review]
nice

**Pull request merged**